packages/hooks/README.md

```diff
@@ -0,0 +1,68 @@
+# Aries Framework Hooks
+
+This package exposes useful hooks that allow you to easily interact with AFJ.
+
+Everything exported from Hooks:
+
+```ts
+import
+	AgentProvider, {
+	useAgent,
+	useConnections,
+	useConnectionById,
+	useConnectionByState,
+	useCredentials,
+	useCredentialById,
+	useCredentialByState,
+	useProofs,
+	useProofById,
+	useProofByState
+} from '@aries-framework/hooks'
+```
+
+First step is to wrap your entire app in our `<AgentProvider/>`. The provider takes two props, the first is your `agentConfig` object and the second is your `genesisUrl`. The base of your app should look something like this:
```

Comment:
As @blu3beri already pointed out, I don't think we should promote the usage of `genesisUrl` as it will download the genesis every time you start the agent. If this is really a desired feature, agents can add this functionality themselves

Reply:
Rather not. It's not part of the feature of this library so we shouldn't include IMO.

Reply:
Can do. 👍 @NeilSMyers could you add this as an item to take care of come Monday?

Reply:
so am I removing the genesisUrl from everything?